Makefile

```diff
@@ -16,15 +16,13 @@ EXTS = $(shell find $(EXT) -type f) \
 	$(shell find $(EXT) -type l)
 SHARE = share
 
-# XXX Make these vars look like git.git/Makefile style
```

Comment:
I was not sure about this comment. Was this just for PREFIX or all the following variables? Please tell me if I deleted it in error.